egs/libri_css/asr1/diarization/VB_diarization.py

```diff
@@ -130,11 +130,11 @@ def VB_diarization(
 
```

Comment:
ditto

Reply:
black formatting issue , it changed few unrelated files , will remove it soon.